com.getferdi.Ferdi.appdata.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.getferdi.Ferdi</id>
```

Comment:
Ideally this should be upstreamed

Reply:
I'll do a PR against upstream for the appdata then. That should not be an issue.

Reply:
Currently the project would have to maintain it. I thought it would be easy to autoupdate the metainfo.xml on build but it doesn't look like its ready yet: https://github.com/electron-userland/electron-builder/issues/1993